crates/sp-domains/src/fraud_proof.rs

```diff
@@ -233,6 +235,20 @@ impl TrueInvalidBundleEntryFraudProof {
             proof_data,
         }
     }
+
+    pub fn matches_with_bundle_entry(&self, inboxed_bundle_entry: &InboxedBundle) -> bool {
```

Comment:
where is this used? Would be nice to add this change to the commit where this was used